build/yamls/antrea-aks.yml

```diff
@@ -3919,9 +3919,20 @@ data:
     #tlsMinVersion:
 
     # The name of the interface on Node which is used for tunneling or routing the traffic across Nodes.
-    # If there are multiple IP addresses configured on the interface, the first one is used.
-    # The interface configured with Node IP is used if this parameter is not set.
+    # If there are multiple IP addresses configured on the interface, the first one is used. The order for
+    # configuring tunneling or routing the traffic across Nodes is (from highest to lowest):
```

Comment:
"The IP address used for tunneling or routing traffic to remote Nodes is decided in the following order of preference:"

Reply:
@jianjuns Thanks for the review, updated.